myoolala/react-starter
# Feat/add lambda api

## Acceptance Criteria
Be able to deploy the complete app via lambda and s3 (serverless)

## Change Description
Adds a new way to build the api in lambda while still developing locally as you would before. Also adds the terraform code needed to be able to build and deploy it in AWS.

## Verification Instructions
- Run the devops init command
- Verify that a new serverless terragrunt folder is made in the AWS folder
- Update the hcl file to reflect your environment
- Deploy the code buckets
- Update the deploy script to reflect your environment and run it
- Cloudfront will take 5ish minutes to build, get a coffee
- Verify that the cloudfront domain on the root path renders the app without crashing
- Verify that the /api/healthcheck, /api/user/active, and POST /api/user/login endpoints all work

## Commit Message
feat: Adds the ability to develop and deploy a 100% serverless version of the app
